lab10.md

```diff
@@ -16,7 +16,11 @@ $ cp -r lab5 lab10 \
 From the Docker documentation: 
 > .. a secret is a blob of data, such as a password, SSH private key, SSL certificate, or another piece of data that should not be transmitted over a network or stored unencrypted in a Dockerfile or in your application’s source code.
 
-This is a more secure alternative to environmental variables. Environmental variables are easier to use but are best suited to non-confidential configuration items.  Seems a good fit for storing the `auth_token` value.  
+This is a more secure alternative to environmental variables. Environmental variables are easier to use but are best suited to non-confidential configuration items, like feature flags or verbose logging.
```

Comment:
I since learned that I was wrong to put `environmental`, can you update to `environment` instead?